lib/_emerge/AbstractPollTask.py

```diff
@@ -93,7 +93,7 @@ def _read_buf(self, fd):
 
     def _async_wait(self):
         self._unregister()
-        super(AbstractPollTask, self)._async_wait()
+        super()._async_wait()
```

Comment:
How did super() work before? (What changed?)

Reply:
I am not sure I understand the details, but this pattern is what's recommended in documentation (which says it's the same as the old pattern):

https://docs.python.org/3.11/library/functions.html#super

Reply:
Oh okay, that's all I needed - thanks!